File: rest-service/src/main/java/mb/lib/formatter/db/Column.java
package mb.lib.formatter.db;

public interface Column
{
  interface FormatJob
  {
    String Config    = "config";
    String CreatedOn = "created_on";
    String JobID     = "job_digest";
    String QueueID   = "queue_id";
    String ReportID  = "report_digest";
    String Status    = "status";
  }

  interface UserLink
  {
    String ReportID    = "report_digest";
    String UserID      = "user_id";
    String Description = "description";
  }
}

File: rest-service/src/main/kotlin/mb/lib/report/db/Column.kt
package mb.lib.report.db

interface Column
{
  object FormatJob
  {
    const val Config    = "config"
    const val CreatedOn = "created_on"
    const val JobID     = "job_digest"
    const val QueueID   = "queue_id"
    const val ReportID  = "report_digest"
    const val Status    = "status"
  }

  object UserLink
  {
    const val ReportID    = "report_digest"
    const val UserID      = "user_id"
    const val Description = "description"
  }
}
